src/editor/C4ConsoleWin32.cpp

```diff
@@ -1139,12 +1139,12 @@ class C4ConsoleGUIPreviewWindow : public C4Window
 		pSurface = new C4Surface(&Application, this);
 	}
 
-	~C4ConsoleGUIPreviewWindow()
+	~C4ConsoleGUIPreviewWindow() override
```

Comment:
Not to me. I guess, inherently, [I sided with clang-tidy](https://bugs.llvm.org//show_bug.cgi?id=30397), [you with the CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rh-override). As far as I can tell, a consensus wasn't reached before the discussion ended on either side. It is left for the programmers to choose their preference (`priority[0] = consistency;`).
Please let me know if you want me to revert these.